### ISSUE TITLE
NVDA does not recognize table row headers in Microsoft PowerPoint

### ISSUE THREAD

**Issue by Tiff-Koval** [nvaccess/nvda#19063]
### Brief summary

The table row headers are not clearly defined for NVDA in PowerPoint. 

### Steps to reproduce

1. Open Microsoft PowerPoint
2. Create a table at least as big as 2X2
3. Fill out each cell 
4. Navigate to table design and ensure "First Column" is checked. Uncheck Header Row if auto checked
5. With NVDA, navigate through the table
6. Notice the table row headers are not recognized as headers

[table rows.pptx](https://github.com/user-attachments/files/22750025/table.rows.pptx)

### Actual behavior

NVDA announces the cells without reference to a header, although defined in the table design. 

"row 2 column 2 "pitbull""

### Expected behavior

The table header rows should be recognized by NVDA.
When navigating cells, the table rows should be associated. It should announce something like
"Dog row 2 Pitbull"

### NVDA type

installed copy

### NVDA version

2025.3

### Windows version

10.0.26100 Build 26100

### Name and version of other software in use when reproducing the issue

PowerPoint Version 2507

### Other information about your system

Other troubleshooting
- When copying and pasting into Microsoft Word, the issue is no longer present
- The issue is not present with VoiceOver or JAWS
- Issue is not present when copying into a PDF

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by gerald-hartig** [comment]
Related to #5350.

**Comment by KD38223** [comment]
I verify this issue is still occurring, same parameters. ctrl + alt + arrow keys does not repeat row header as it does in Microsoft Word.
